linode/manager
# upcoming: [M3-7455] - Add child account access column and disable button when account has child accounts

## Description 📝
Add child account access column and disable button when account has child accounts.

## Changes  🔄
List any change relevant to the reviewer.
- Add child account access column
- Disable button when account has child accounts

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-12-27 at 3 29 43 PM](https://github.com/linode/manager/assets/139489745/806df3d4-e388-4dec-93b8-94f2aa680b5f) | ![Screenshot 2023-12-27 at 3 45 56 PM](https://github.com/linode/manager/assets/139489745/b6a88d64-ebcd-411a-86c0-6113f38b58f6) |
| ![Screenshot 2023-12-27 at 3 30 15 PM](https://github.com/linode/manager/assets/139489745/d1270e5c-8b4a-44f0-a2da-4a82bfb9aea4) | ![Screenshot 2023-12-27 at 3 30 45 PM](https://github.com/linode/manager/assets/139489745/64b3c71d-8442-436a-b0cb-2a0c6ebb0a4c) |

## How to test 🧪

### Prerequisites
- MSW on
- Parent/Child Account flag on

### Verification steps 
- Navigate to http://localhost:3000/account/users and ensure there is an additional 'Child Account Access' column with accurate labels ('Enabled' and 'Disabled')
- Navigate to http://localhost:3000/account/settings and ensure the button is disabled since the current account has a child account, and ensure there is a notice explaining why it is disabled.
- Ensure the parent/child feature flag works as expected

## As an Author I have considered 🤔

*Check all that apply*
